### DISCUSSION TITLE
Choose source site when adding new site

### DISCUSSION THREAD

**Discussion by dowadidi** [craftcms/cms#11815] [Ideas] [answered]
When adding a new site to an existing setup, content is being copied to populate the new site. Documentation is not clear on which site is being copied. 

It would be nice to have the option to choose the site you want the content of the new site to be based on.

**Comment by brandonkelly** [comment, marked as answer]
Content is copied from the primary site. Agree it would be nice to be able to override that, however I’m not totally sure how exactly we’d do that in a way that would also work when deploying to production. We’d perhaps need to start storing the source site in the site’s project config, basically in perpetuity, which feels a little weird.
